packages/component-library/src/components/plmg-radio-button-group/plmg-radio-button-group.scss

```diff
@@ -0,0 +1,37 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Component style
+// -------------------------------------------------
+fieldset {
```

Comment:
again, wrap everything into a custom class name: `.plmg-radio-button-group`.

If it was a shadow component, all styles would be encapsulated and it would be nice. Since it's not a shadow component (and it must not be ;) ) then you need a custom class name with the prefix `pmlg-` to avoid naming collision.

Oopsie every span is red :see_no_evil: 
![image](https://user-images.githubusercontent.com/55230735/169557036-3220b2df-d294-4339-b677-2bbfa0e1ed1c.png)

Reply:
Wops! Nice catches! How come we need the plmg prefix to avoid naming collision? Or is that more convention?

Reply:
More of a convention :)